docs/source/conf.py

```diff
@@ -189,10 +189,6 @@ def setup(app):
     app.connect("autodoc-skip-member", skip)
 """
 
-# @jpchen's hack to get rtd builder to install pytorch
+# @jpchen's hack to get rtd builder to install latest pytorch
 if 'READTHEDOCS' in os.environ:
-    os.system('pip install awscli')
-    os.system('aws s3 --no-sign-request sync s3://pyro-ppl/ci tmp --exclude "*" --include "*-cp27-*";')
-    os.system('pip install tmp/*')
-    os.system('rm -r tmp')
-    os.system('pip install -e .')
+    os.system('sh scripts/install_pytorch.sh')
```

Comment:
If this doesn't work, it could be because the installation script assumes bash and I'm not sure what `sh` points to here.